browser-extension/api-server/Dockerfile

```diff
@@ -9,6 +9,4 @@ RUN npm install -g nodemon
 COPY ./entrypoint.sh /
 RUN chmod +x /entrypoint.sh
 ENTRYPOINT ["/entrypoint.sh"]
-COPY . .
```

Comment:
@duggalsu, hey are you sure this doesn't break the `api-server`. If the source code of the server is not copied, then this pretty much means there would be nothing to run. If you run node index.js, there wont be any index.js file even.
Its possible that if you tested this, you had stale files from before. If you did a fresh clone of the repo and tried to do `docker-compose up`, i bet `api-server` service will fail.

This might need closer inspection and testing.

Reply:
I deleted the docker images and `node_modules` manually and tested successfully. Will test with a fresh clone of the repo too.

Reply:
I ran the following command to remove existing docker images
```
docker rmi browser-extension-api-server mysql adminer
```

Did a fresh clone of Uli and ran all the steps. The `api-server` is running and the console shows an update message when the slur list is updated.

This seems to be working correctly on my end.

Reply:
very interesting. I'll believe it when i see it :D